### PR TITLE
Add iOS changelog to explicit animation deactivation

### PR DESCRIPTION
**Pull request by sgara**

### CODE REVIEW

File: _pages/iOS/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+{% assign versions = site.data.versions[page.platform] %}
+
+* Placeholder for Table of Content
+{:toc}
+
+# Version 2.0.0
+
+## Added
+- ARKit tracking to all compatible devices
```
Comment (mmorisset) [inline _pages/iOS/CHANGELOG.md]: ARKit tracking strategy to all compatible devices

File: _pages/iOS/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Fallback to Vuforia for non-ARKit capable devices
```
Comment (mmorisset) [inline _pages/iOS/CHANGELOG.md]: Gyroscope and universal tracker fallback for non-ARKit capable devices

File: _pages/iOS/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- Switch between ARKit and Vuforia is automatically selected, based on device capabilities
```
Comment (mmorisset) [inline _pages/iOS/CHANGELOG.md]: To remove

File: _pages/iOS/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- Tracking events callback
```
Comment (mmorisset) [inline _pages/iOS/CHANGELOG.md]: Tracking strategies callbacks

File: _pages/iOS/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Migrating to v1.X to v2.0
+Migration is seemless, you just have to update AugmentPlayerSDK pod version.
+
+**Note:** Triggered animations and animation scenarios are not activated while using
+ARKit system. No animation is played in this mode.
+Triggered animations and animation scenarios are still activated when using Vuforia.
```
Comment (mmorisset) [inline _pages/iOS/CHANGELOG.md]: Triggered animations and animation scenarios are still activated when using gyroscope and tracker strategies.

**Comment by sgara** [comment]
@mmorisset updated

**Pull request merged**